ansibullbot/triagers/ansible.py

```diff
@@ -851,6 +851,7 @@ def create_actions(self, iw, actions):
                 actions.cancel_ci_branch = True
                 return
 
+            a = self.is_bot()
```

Comment:
```
  File "/home/johnb/git/work/ansible-community/collection_bot/ansibullbot/triagers/ansible.py", line 854, in create_actions
    a = self.is_bot()
AttributeError: 'AnsibleTriage' object has no attribute 'is_bot'
```

Reply:
`AnsibleTriage` is sub-classing `DefaultTriager` which is an object:
https://github.com/ansible-community/collection_bot/blob/eaf73eb107d0ce0706e9c6f540067fbbadd3ce82/ansibullbot/triagers/defaulttriager.py#L84

But you added `is_bot` to `DefaultWrapper`?